saml.go

```diff
@@ -0,0 +1,92 @@
+package tfe
+
+import (
+	"context"
+)
+
+// Compile-time proof of interface implementation.
+var _ Saml = (*saml)(nil)
+
+// Saml describes SAML settings
+// TFE API docs: https://www.terraform.io/docs/cloud/api/agents.html
```

Comment:
Did you mean to put this URL? This isn't the SAML Settings docs: https://www.terraform.io/docs/cloud/api/admin/settings.html

Reply:
Ah yep, copy paste error there, let me fix that